Setup/docker-compose_dev.yml

```diff
@@ -4,29 +4,56 @@ version: "3.3"
 # YML file for Dev
 
 services:
-  collector:
-     image: securebrowsing/shield-collector:latest
-     hostname: collector
-     volumes:
-       - /var/run/docker.sock:/var/run/docker.sock
-       - /proc:/hostfs/proc:ro
-       - /sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro
-       - /:/hostfs:ro
-     networks:
-       - shield
-     deploy:
-       mode: global
-       restart_policy:
-           condition: any
-       update_config:
-           failure_action: rollback
-       resources:
-         limits:
-           memory: 2GB
-           cpus: "1"
-     environment:
-       - "SYSTEM_PERIOD=5m"
-       - "DOCKER_PERIOD=5m"
+#  collector:
+#     image: securebrowsing/shield-collector:latest
+#     hostname: collector
+#     volumes:
+#       - /var/run/docker.sock:/var/run/docker.sock
+#       - /proc:/hostfs/proc:ro
+#       - /sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro
+#       - /:/hostfs:ro
+#     networks:
+#       - shield
+#     deploy:
+#       mode: global
+#       restart_policy:
+#           condition: any
+#       update_config:
+#           failure_action: rollback
+#       resources:
+#         limits:
+#           memory: 2GB
+#           cpus: "1"
+#     environment:
+#       - "SYSTEM_PERIOD=5m"
+#       - "DOCKER_PERIOD=5m"
+
+
+  netdata:
+    image: titpetric/netdata:latest
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    networks:
+      - shield
+    hostname: netdata
+    ports:
+      - 8282:19999
+    cap_add:
+      - SYS_PTRACE
+    volumes:
+      - /proc:/host/proc:ro
```

Comment:
You have mount proc directory twice it's broke installation.
I fix it. See my commit later